wpilibc/athena/src/Joystick.cpp

```diff
@@ -279,10 +279,9 @@ Joystick::HIDType Joystick::GetType() const {
  */
 std::string Joystick::GetName() const { return m_ds.GetJoystickName(m_port); }
 
-// int Joystick::GetAxisType(uint8_t axis) const
-//{
-//  return m_ds.GetJoystickAxisType(m_port, axis);
-//}
+int32_t Joystick::GetAxisType(uint8_t axis) const {
+  return m_ds.GetJoystickAxisType(m_port, axis);
+}
```

Comment:
If I remember right it being commented out was basically a reminder to fix it/figure out why it caused crashes. We wanted to get the other working functionality in and I didn't have time to figure out what was going on with this one method.

Definitely should have had a TODO: comment added